ros1_rosbag_storage_vendor/cmake/CMakeLists.txt.in

```diff
@@ -0,0 +1,4 @@
+# rosbag_storage does not contain a toplevel CMakeLists so colcon gets confused
+# This is also an opportunity, because we want to only build rosbag_storage anyways
```

Comment:
I am not sure I understand this line. What else would we build if not only rosbag_storage?

Reply:
The whole of `ros_comm`. Note that we checkout `ros_comm`, which contains quite a few different packages. If we tried to build the package directly by moving the working directory around, it would try to also build all the other packages in `ros_comm` and then fail because those are catkin packages, so we'd have to add some COLCON_IGNOREs to all other directories.

Reply:
Should be changed.